content/blog/2022/03/admission-policy.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Introducing the AdmissionPolicy
+authors:
+- Raul Cabello Martin
+date: 2022-03-09
+---
+
+Up until now, the only way to define a policy in Kubewarden was to use a `ClusterAdmissionPolicy` which is applied to cluster-wide resources across all namespaces.
+
+That's why we're thrilled to announce the new `AdmissionPolicy` resource. This new resource is created inside a `namespace` and the policies will process only the requests that are targeting the namespace where the `AdmissionPolicy` is defined. Except from being a "namespaced" resource, `AdmissionPolicy` works exactly the same as the `ClusterAdmissionPolicy`.
+
+## Understanding the motivation for using AdmissionPolicies
+
+It's possible to restrict the namespaces where a `ClusterAdmissionPolicy` evaluates resources using a `namespaceSelector`, however there's no way for Kubernetes administrators to restrict users from creating `ClusterAdmissionPolicies` that evaluate resources just in a particular namespace. 
+
+Allowing all tenants to deploy `ClusterAdmissionPolicies` is risky. A tenant could apply policies that affect resources in all namespaces, even if they don't have access to all of them.
+You probably want to allow tenants to deploy policies just in the namespaces they have access to. That's where `AdmissionPolicies` come into play! We can achieve this restriction with `AdmissionPolicies` and Kubernetes’ Role-Based Access Control (RBAC).
+
+As an example, let's say you have a tenant who can deploy resources just to the `development` namespace. You can allow this tenant to deploy `AdmissionPolicies` just in this namespace using RBAC. If you allowed them to create `ClusterAdmissionPolicies`, they could block other resources from being handled in other namespaces.
```

Comment:
Since we’re postulating that the tenant should be able to deploy only to the development namespace, slightly rephrasing the existing sentence.

```suggestion
As an example, let's say you want a tenant who can deploy resources just to the `development` namespace. You can allow this tenant to deploy `AdmissionPolicies` just in this namespace using RBAC. However, if you allowed them to create `ClusterAdmissionPolicies`, they could block other resources from being handled in other namespaces.
```